domain/src/main/java/com/yapp/itemfinder/domain/repository/LockerRepository.kt

```diff
@@ -1,7 +1,7 @@
 package com.yapp.itemfinder.domain.repository
 
-import com.yapp.itemfinder.domain.model.Data
+import com.yapp.itemfinder.domain.model.Locker
 
 interface LockerRepository {
-    suspend fun getAllLockers(): List<Data>
+    suspend fun getLockers(spaceId: Long): List<Locker>
```

Comment:
`Entity`를 suffix로 붙이는 게 좋을 것 같습니다.
```suggestion
    suspend fun getLockers(spaceId: Long): List<LockerEntity>
```

Reply:
수정했습니다